cs-couchbase/src/main/java/io/cloudslang/content/couchbase/utils/InputsUtil.java

```diff
@@ -24,6 +33,11 @@
  */
 
 public class InputsUtil {
+    private static final String ALLOW_ALL = "allow_all";
```

Comment:
why don't you add this in the Inputs class 
(cs-couchbase/src/main/java/io/cloudslang/content/couchbase/entities/constants/Inputs.java)
or somewhere more fit ?